src/BizHawk.Client.Common/savestates/ZipStateLoader.cs

```diff
@@ -9,7 +9,8 @@ namespace BizHawk.Client.Common
 	public class ZipStateLoader : IDisposable
 	{
 		private ZipArchive _zip;
-		private Version _ver;
+		private Version _zipVer;
+		private Version _stateVer;
```

Comment:
Someone used `Version` here initially, which is definitely not the right class to use. Doesn't an `int` suffice?
That would probably make the entire logic simpler and easier to understand as well.